packages/synthetic-chain/src/cli/dockerfileGen.ts

```diff
@@ -226,7 +230,10 @@ export function writeDockerfile(
       `#----------------\n# ${proposal.proposalName}\n#----------------`,
     );
 
-    if (proposal.type === '/agoric.swingset.CoreEvalProposal') {
+    if (
```

Comment:
good time to make this a `switch`. I don't recall why I didn't do that initially. Maybe in early stages there were variable assignments.

Reply:
as we discussed, I'll leave it to you to wrap this PR up